gpu-enabler/src/main/scala/com/ibm/gpuenabler/CUDARDDUtils.scala

```diff
@@ -141,7 +142,7 @@ private[gpuenabler] class ConvertGPUPartitionsRDD[T: ClassTag](
         hyIter
       }
       case iter: Iterator[T] => {
-        // println("Converting Regular Iterator to hybridIterator")
+    	// println("Converting Regular Iterator to hybridIterator")
```

Comment:
nit: can we revert this change?

Reply:
Yes.I mistakenly touched it.

Reply:
nit: Would it be possible to revert this change?